src/Transports/MassTransit.ActiveMqTransport/Topology/Topologies/ActiveMqMessageConsumeTopology.cs

```diff
@@ -27,7 +27,11 @@ public ActiveMqMessageConsumeTopology(IMessageTopology<TMessage> messageTopology
             _messageTopology = messageTopology;
             _publishTopology = publishTopology;
 
-            _consumerName = $"VirtualTopic.{messageTopology.EntityName}::Consumer.{{queue}}.VirtualTopic.{messageTopology.EntityName}";
+            _consumerName = $"Consumer.{{queue}}.VirtualTopic.{messageTopology.EntityName}";
+            if (ActiveMqArtemisSupport.EnableArtemisVirtualTopicNamingSupport)

```

Comment:
So this should be configured on the IActiveMqHostTopology, and used by the public/consume topologies.